.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -56,95 +56,22 @@ steps:
-      set -e
-      m="windowsVhdMode"
-      echo "SIG_GALLERY_NAME: ${SIG_GALLERY_NAME}"
-      echo "SIG_IMAGE_NAME_PREFIX: ${SIG_IMAGE_NAME_PREFIX}"
-      echo "SIG_IMAGE_VERSION: ${SIG_IMAGE_VERSION}"
-      
-      if [[ -n ${SIG_GALLERY_NAME} && -n ${SIG_IMAGE_NAME_PREFIX} && -n ${SIG_IMAGE_VERSION} ]]; then
-          echo "Non prod build"
-          sigImageName="${SIG_IMAGE_NAME_PREFIX}-${WINDOWS_SKU}"
-          echo "##vso[task.setvariable variable=SIG_FOR_PRODUCTION]False"
-          echo "##vso[task.setvariable variable=SKIPVALIDATEREOFFERUPDATE]True"
-      else
-          echo "prod build"
-          sigImageName="windows-${WINDOWS_SKU}-$(date +"%H%M%S")-$RANDOM"
-          WS_SKU=$(echo $WINDOWS_SKU | tr '-' '_')
-          sigGalleryName="WSGallery$(date +"%y%m%d")"
-          echo "##vso[task.setvariable variable=SIG_GALLERY_NAME]$sigGalleryName"
-          echo "##vso[task.setvariable variable=SIG_IMAGE_VERSION]1.0.0"
-          echo "##vso[task.setvariable variable=SIG_FOR_PRODUCTION]True"
-          echo "##vso[task.setvariable variable=SKIPVALIDATEREOFFERUPDATE]True"
-      fi
-      echo "Set build mode to $m"
-      echo "##vso[task.setvariable variable=SIG_IMAGE_NAME]$sigImageName"
-      echo "##vso[task.setvariable variable=MODE]$m"
-      if [[ "${USE_RELEASE_DATE}" = "False" ]]; then 
-        echo "use current date as build date";  BUILD_DATE=$(date +"%y%m%d")
-      else 
-        echo "use release date as build date"
-        echo "${RELEASE_DATE}" | grep -E '[[:digit:]]{6}'
-        if (( $? != 0 )); then
-          echo "The release date ${RELEASE_DATE} is not valid date. Release date format: YYMMDD."
-          exit 1
-        fi
-        BUILD_DATE=${RELEASE_DATE}
-      fi 
-      echo "Default BUILD_DATE is $BUILD_DATE" 
-      if [[ -n "${CUSTOM_BUILD_DATE}" ]]; then 
-        echo "set BUILD_DATE to ${CUSTOM_BUILD_DATE}"
-        BUILD_DATE=${CUSTOM_BUILD_DATE}
-      fi
-      echo "Set build date to $BUILD_DATE" 
-      echo "##vso[task.setvariable variable=BUILD_DATE]$BUILD_DATE" 
-    displayName: Set pipeline variables
+  - bash: make -f packer.mk az-login
+    displayName: AZ Login
     env:
-      WINDOWS_SKU: ${{ parameters.windowsSku }}
+      MODE: "windowsVhdMode"
+      SUBSCRIPTION_ID: $(SUBSCRIPTION_ID)
 
-  - script: |
-      # because SUBSCRIPTION_ID is defined as a pipeline variable, setting this in the "env" section below doesn't work.
-      export SUBSCRIPTION_ID=$(AZURE_BUILD_SUBSCRIPTION_ID)
-      make -f packer.mk run-packer-windows
-    displayName: Building windows VHD
+  - bash: bash ./.pipelines/scripts/windows_build_vhd.sh
+    displayName: Build and save VHD
     env:
+      WINDOWS_SKU: ${{ parameters.windowsSku }}
       BRANCH: $(Build.SourceBranch)
-      MODE: $(MODE)
       POOL_NAME: $(AZURE_POOL_NAME)
-      SUBSCRIPTION_ID: $(AZURE_BUILD_SUBSCRIPTION_ID)
-      AZURE_RESOURCE_GROUP_NAME: $(AZURE_BUILD_RESOURCE_GROUP_NAME)
+      SUBSCRIPTION_ID: $(SUBSCRIPTION_ID)
```

Comment:
Should we qualify SUBSCRIPTION_ID like the others?  It seems to be set from AZURE_BUILD_SUBSCRIPTION_ID above, but I see below we also have AZURE_BUILD_SUBSCRIPTION_ID so unsure what it refers to.

Reply:
And changing their values in the variables attached to the pipeline could break other pipelines, so I was hesitant to do that. Might do it as a cleanup task after this PR is merged and everything is working again.